modules/media.js

```diff
@@ -75,7 +78,14 @@ Media.prototype.clean = function(type, oncomplete) {
   }, function(error) {
     if (error) {
       errors.push(error);
+    } else {
+      ffosbr.history.set(type, {
+        title: capitalize(type),
```

Comment:
Yes this is a safe generalization because media.clean and media.backup are only called on the media types.